spec/worker-helpers-spec.js

```diff
@@ -56,6 +56,22 @@ describe('Worker Helpers', () => {
       expect(foundEslint.path).toEqual(expectedBundledPath)
       expect(foundEslint.type).toEqual('bundled fallback')
     })
+
+    it('should handle a null projectPath when resolving path', () => {
+      const modulesDir = Path.join(getFixturesPath('local-eslint'), 'node_modules')
+      const config = {
+        advancedLocalNodeModules: '../node_modules',
+        useGlobalEslint: false
+      }
+
+      const findEslint = () => Helpers.findESLintDirectory(modulesDir, config, null)
```

Comment:
Explicitly calling it with `null` here supposedly replicates what you are seeing, but I'd rather this test replicate the actual situation that triggers this case, not forcing it like this.

The tests should always replicate the real world case where possible.